test/software/amazon/ion/GoodIonTest.java

```diff
@@ -88,17 +89,10 @@ public void test()
             in.close();
         }
 
-
-        // Pass 4: Encode to binary, and use Reader
-        if (! myFileIsBinary) {
-            // Check the encoding of text to binary.
             treeReader = system().newReader(datagram);
```

Comment:
It looks like when the conditional was removed, the contained lines weren't un-indented.